test_tf2/CHANGELOG.rst

```diff
@@ -97,7 +97,7 @@ Changelog for package test_tf2
 ------------------
 * splitting rospy dependency into tf2_py so tf2 is pure c++ library.
 * switching to console_bridge from rosconsole
-* moving convert methods back into tf2 because it does not have any ros dependencies beyond ros::Time which is already a dependency of tf2
+* moving convert methods back into tf2 because it does not have any ros dependencies beyond builtin_interfaces::msg::Time which is already a dependency of tf2
```

Comment:
To aggressive global search and replace. Reverted here and in a few other places.